package.json

```diff
@@ -47,33 +52,37 @@
   "dependencies": {
     "arrify": "^2.0.1",
     "micromatch": "^4.0.2",
-    "schema-utils": "^2.6.6"
+    "schema-utils": "^2.7.0",
+    "@types/stylelint": "^9.10.1"
```

Comment:
I think `stylelint` also can be moved `to` `devDependencies`. Why? Because we have
- `stylelint` in peer deps and stylelint provide types out of box https://github.com/stylelint/stylelint/tree/master/types

Reply:
Now yes, lol